src/content/release-notes.mdx

```diff
@@ -5,6 +5,38 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.26.0
+
+11 November 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- Hostname Length Limit: Deploys will now fail if a service generates a hostname longer than 63 characters, providing an error message to guide users. Previously, the endpoint wouldn’t work, but the dev environment still deployed successfully. This change may affect environments that previously deployed without issue.
+- Helm Release Name Limit: We are limiting the length of Helm release names and allowing to override the default backend service name to make it shorter if needed. The defaultBackend.nameOverride allows to change the full name of the default backend service. In case you want to change it as part of an upgrade, it might have side effects in the installation, so please, follow this guide
+- Deploying Private Repositories now uses the Okteto backend as the SSH agent instead of mounting the local SSH agent into BuildKit. This change provides feature parity between remote and local deploys but may affect cases where repositories are cloned as part of a command defined in your Okteto Manifest.
+- Buildkit Persistence is now enabled by default. The default disk size is set to 100Gi and cache size is now specified as a ratio of the total disk size (default: 0.9). For example, with a 100Gi disk, 90% will be allocated to cache. If you had previously set `buildkit.persistence.cache setting` please adjust the new ratio accordingly, as that setting will not apply any longer
+
+### New Features
+
+- **Introducing the Okteto Resource Manager**: a new feature that automatically optimizes CPU and memory requests for your environments. By analyzing real-time resource utilization, the Resource Manager dynamically adjusts resource requests to ensure efficient usage, prevent node overload, and improve overall cluster performance. This feature simplifies resource management, reduces manual adjustments, and enhances application stability, especially in larger clusters
+- Added Okteto Garbage Collector settings to the Admin Dashboard: Admins of Okteto can now manage sleep and delete periods from the Dashboard
+- Added additional feature and documentation for [running Buildkit at scale](self-hosted/manage/buildkit-high-performance.mdx)  <!-- 8727 -->
+
+### Improvements
+
+- Create docker secret if one doesn't already exist in the controller manager  <!-- 8566 -->
+- Buildkit cache size is now automatically configured based on its PVC volume size  <!-- 8723 -->
+- Removed buildkit persistency as an installation step now that it defaults to true  <!-- 8771 -->
+- We now allow the Okteto frontend and Buildkit manifest with flags and properties to run rootless   <!-- 8774, 8800 -->
```

Comment:
- Frontend always run as rootless and it's not configurable
- Buildkit, if rootless is enabled, doesn't use `privilege`